application/models/post.go

```diff
@@ -4,9 +4,10 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/gobuffalo/buffalo"
+	"github.com/gobuffalo/events"
 	"time"
 
-	"github.com/gobuffalo/buffalo"
 	"github.com/gobuffalo/nulls"
```

Comment:
Are you manually adding imports? `goimports` sorts differently than this.

Reply:
sorry, I got a merge conflict and must have chosen the wrong order.

Reply:
Oh, that makes sense. No problem.

Reply:
I think I got it corrected in the latest push